jsons/deserializers/default_complex.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict
+
+
+def default_complex_deserializer(obj: Dict[str, float],
+                                 cls: type = complex,
+                                 **kwargs) -> complex:
+    """
+    Deserialize a dictionary with 'real' and 'imag' keys to a complex number.
+    :param obj: the dict that is to be deserialized.
+    :param cls: not used.
+    :param kwargs: not used.
+    :return: an instance of ``complex``.
+    """
+    real = obj.get('real')
```

Comment:
You could reuse the parsing and checking of the other ``jsons`` deserializers, like so:

```python
    try:
        clean_obj = load({'real': obj['real'], 'imag': obj['imag']},
                         cls=Dict[str, float])
        return complex(clean_obj['real'], clean_obj['imag'])
    except KeyError as err:
        raise AttributeError("Cannot deserialize {} to a complex number, "
                             "does not contain key '{}'"
                             .format(obj, err.args[0]))
    except DeserializationError as err:
        raise AttributeError("Cannot deserialize {} to a complex number, "
                             "cannot cast value {} to float"
                             .format(obj, err.source))
```

Reply:
Thanks, I've changed this :)